pangeo_forge_esgf/dynamic_kwargs.py

```diff
@@ -13,21 +13,35 @@
     # TODO: I might not want to allow this in the ocean and ice fields. Lets see
 )
 
+daily_divisors = sorted(
+    [1, 30, 360, 365, 3653]
+    # TODO: It  might be more efficient to default to the underlying file chunks?
```

Comment:
I dont think so. Dealing with the underlying chunks, and outputting them in a way that makes sense to the user is one of the core tasks pangeo-forge aims to solve.  These divisons should rather reflect scientifically useful units to e.g. avoid splitting e.g. a yearly chunk with a few days left, as this would make certain operations (annual average) much slower to compute. 
These look good for now, but I was wondering about different calendars and how we could maybe influence which divisors to use depending on the calendar of the data. 
Just thinking out loud here.

Reply:
Yeah, that makes sense. As far as I know there isn't any ESGF metadata to describe the calendar though and we'd have to open the files to inspect the time coordinate metadata directly which would be pain.

Reply:
Yeah I think this is a problem to be pushed to later work.